### ISSUE TITLE
[BUG] `nvm-desktop: command not found: "node"` after installing first Node version from GUI without running `nvmd use {version}`

### ISSUE THREAD

**Issue by iki** [1111mp/nvm-desktop#128]
**Describe the bug**

After fresh install of NVM Desktop and install of first Node version from GUI, default version is not set in `%UserProfile%\.nvmd\default` (file does not exist) and all node/npm/npx/corepack shims exit with `nvm-desktop: command not found: "node"` (replace node with npm/npx/corepack)

**To Reproduce**

1. Install `NVM.Desktop_4.0.1_x64-setup.exe` on Windows 11 x64
2. Install Node v22.9.0 via NVM Desktop
3. **[OK]** Optionally check `%UserProfile%\.nvmd\bin` is in PATH
4. **[OK]** Optionally check Node v22.9.0 is installed in `%UserProfile%\.nvmd\versions\22.9.0`
5. **[BUG]** Check `%UserProfile%\.nvmd\default` -> **Missing**
6. **[BUG]** Try running node/npm/npx/corepack via nvmd shim. Here I use explicit path to be sure:
```sh
$ "%UserProfile%\.nvmd\bin\node.exe" --version
nvm-desktop: command not found: "node"
# Same for npm/npx/corepack
```
7. **[Workaround]** Run `nvmd use` explicitly
```sh
$ nvmd use v22.9.0
Now using node v22.9.0
```
8. **[OK]** Repeat step 6 to verify shims work OK now.

**Expected behavior**
- NVM Desktop GUI install of first Node version should set it as default
- Error `nvm-desktop: command not found: "node"` should be more explicit: include where nmvd searched for it
- If default is missing, it should report it, e.g. `nvm-desktop: default node version is not set, use 'nvmd use {version}`

**Desktop (please complete the following information):**
 - OS: Windows 11 23H2 Home 10.0.22635.4367
 - Arch: x64
 - NVM Desktop Version: 4.0.1

**Comment by 1111mp** [comment]
First of all, thank you for your feedback. 🌹 

Currently, after downloading a version, you need to manually set it as the default version through the `Apply` button:
![image](https://github.com/user-attachments/assets/6391205c-6c98-4c3c-b1e4-da741a80bfef)

> NVM Desktop GUI install of first Node version should set it as default

I think doing so will make some users feel unacceptable (a feeling of being forced to take over). Perhaps there is another way to solve this problem: add a checkbox in the download pop-up box to "set it as the default version". If checked, it will be set as the default version after downloading, otherwise it will do nothing. Or maybe you have a better idea.

> - Error nvm-desktop: command not found: "node" should be more explicit: include where nmvd searched for it
> - If default is missing, it should report it, e.g. nvm-desktop: default node version is not set, use 'nvmd use {version}

Yes, a clearer and more complete error message should be given. I will improve this part in the next minor version.

**Comment by 1111mp** [comment]
This issue has been fixed in [Release v4.0.2](https://github.com/1111mp/nvm-desktop/releases/tag/v4.0.2). You can upgrade to this version to verify it. Thanks. 🌹

**Comment by iki** [comment]
Works perfectly! Many thanks!
